### PR TITLE
hive-frontend ClusterRoleBinding fix

### PR DESCRIPTION
**Pull request by maorfr**
attempts to fix #715

### CODE REVIEW

File: config/rbac/hive_frontend_role_binding.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: hive-frontend
+groupNames:
```
Comment (staebler) [inline config/rbac/hive_frontend_role_binding.yaml]: This is not a valid field for ClusterRoleBinding in rbac.authorization.k8s.io/v1.

Comment (staebler) [reply config/rbac/hive_frontend_role_binding.yaml]: This is a field from ClusterRoleBinding in authorization.openshift.io/v1, which is not what is being used here. Even in that, the field is legacy and has been superseded by the use of a Group subject, as was done in #715.

**Comment by dgoodwin** [comment]
Just as an FYI there's some really funky stuff going on with the auth types where they're aliased and not separate types. I can't remember all the details but there's some weird magic going on behind the scenes.

**Comment by maorfr** [comment]
i'll look into this again tomorrow morning. thanks!

File: config/rbac/hive_frontend_role_binding.yaml (hunk 1 above)
Comment (maorfr) [reply config/rbac/hive_frontend_role_binding.yaml]: submitted a fix in #719

**Pull request merged**